requirements.txt

```diff
@@ -0,0 +1,2 @@
+Django==2.0
```

Comment:
This version identifier will always install Django 2.0 specifically - we should either specify the exact version we want (`Django==2.0.5`), or a compatible release specifier so we get the latest minor update (`Django~=2.0`) - see the [PEP on version specifiers](https://www.python.org/dev/peps/pep-0440/#version-specifiers).

Reply:
No problem. Updated to be more specific in commit `b58aca6`.